fossunited/fossunited/doctype/foss_event_cfp_submission/templates/foss_event_cfp_submission.html

```diff
@@ -9,7 +9,7 @@
             <span>{{ event.event_name }}</span>
         </a>
         <i class="ti ti-chevron-right"></i>
-        <a href="/{{event.route}}/cfp/all">
```

Comment:
A redirect for /all page will ensure that people can share it easily. 

`/event/xyz/cfp/all` just seems natural in contrast to `/dashboard/cfp/xyz`.

Moreover, people are already used to /all now. Can't take that away from people

Reply:
People already are used to /all, wouldn't changing that now be too confusing without any warning whatsoever?

A simple redirect code is the easiest solution to tackle this. 

I don't think a rewrite in vue is going to happen ever, because of the complexities with 2 vue folders and routings associated with it. 

let's just add `/all` redirect, there are no downsides to it

Reply:
> A simple redirect code is the easiest solution to tackle this.

It is the easiest, but IMO not the best. And definitely not a something we could follow, as the process itself generates unused files into the codebase. There should be a better alternative.

I will add this patch on for now, due to time crunch. But will take suggestions for work-arounds.